fbw-a380x/src/systems/systems-host/systems/LegacyFuel.ts

```diff
@@ -1,4 +1,4 @@
-import { MathUtils } from '@flybywiresim/fbw-sdk';
+import { MathUtils, UpdateThrottler } from '@flybywiresim/fbw-sdk';
```

Comment:
UpdateThrottler should be deprecated in favour of using ``ClockEvents`` i.e. 

```

    const sub = this.bus.getSubscriber<ClockEvents>();
    let lastUpdateTime: number;
    sub
      .on('realTime')
      .onlyAfter(1000)
      .handle((now) => {
        const dt = lastUpdateTime === undefined ? 0 : now - lastUpdateTime;
        lastUpdateTime = now;
        this.update(dt);
      });
```

Add a TODO FIXME: for this

Reply:
This is not relevant anymore, is it?